variables.tf

```
@@ -184,3 +184,22 @@ variable "build_environment_variables" {
   default     = {}
   description = "A set of key/value environment variable pairs available during build time."
 }
+
+variable "docker_registry" {
+  type        = string
+  default     = "CONTAINER_REGISTRY"
+  description = "Docker Registry to use for storing the function's Docker images. Allowed values are CONTAINER_REGISTRY (default) and ARTIFACT_REGISTRY."
+}
+
+variable "docker_repository" {
+  type        = string
+  default     = ""
```

Would this be better as `null`?

```suggestion
  default     = null
```